### PR TITLE
Add `pressF4` and `pressF1` methods to GridView and TextField

### PR DESCRIPTION
**Pull request by wisenrico**
These two methods are missing on the SapGUILibrary.

### CODE REVIEW

File: packages/main/src/RPA/SAP.py [hunk 1]
```diff
@@ -193,3 +193,29 @@ def get_cell_value(self, table_id, row_num, col_id):
             self.take_screenshot()
             message = "Cannot find Column_id '%s'." % col_id
             raise ValueError(message) from ex
+    
+    def press_f1(self, element_id):
```
Comment (mikahanninen) [inline packages/main/src/RPA/SAP.py:197]: Can you add the type into the method parameters ? `element_id: str`

File: packages/main/src/RPA/SAP.py [hunk 2]
```diff
@@ -193,3 +193,29 @@ def get_cell_value(self, table_id, row_num, col_id):
...
+        """Calls the PressF1 function on the passed element id.
+        The element can be a textfield or a GridView inside a shell.
+        
+        :param element: SAP element
```
Comment (mikahanninen) [inline packages/main/src/RPA/SAP.py:201]: param `element` needs to match the parameter name which is `element_id`

**Review by mikahanninen — CHANGES_REQUESTED** [review]

**Comment by mikahanninen** [comment]
Hi @wisenrico! Thank you for the contribution ❤️ I added couple of comments.

**Comment by wisenrico** [comment]
I have addressed the comments 👍🏻

**Comment by mikahanninen** [comment]
You also need to accept the Github's [CLA](https://cla-assistant.io/robocorp/rpaframework?pullRequest=1130).

**Review by mikahanninen — APPROVED** [review]

**Pull request merged**